code/index.html

```diff
@@ -3,28 +3,52 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Holmsjö</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
+    <header>
+      <section class="header">
+        <h1>Holmsjö</h1>   
+      </section>
+    </header>
+    <main class="main">
+        <!-- Form -->
+      <section class="signup-form">
+        <h2>Sign up for Newsletter</h2>
+        <form action="https://httpbin.org/anything" method="POST">
+          <label>
+           <input type="text" placeholder="First name" name="firstname" required>
+          </label>
+           <label>
+            <input type="text" placeholder="Last name" name="lastname" required>
+          </label>
+           <label>
+            <input type="text" placeholder="Enter your email" name="email" required>
```

Comment:
There's a type called email that will help you with validation! What it means is that your browser will look and see if it is a email that you enter. You most have `xx@xx.xx` to be able to post!
```suggestion
            <input type="email" placeholder="Enter your email" name="email" required>
```